cmd/ferretdb/main_windows.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build windows
+
+package main
+
+import (
+	"context"
+	"os/signal"
+
+	"golang.org/x/sys/windows"
+)
+
+func notifyAppTermination(parent context.Context) (context.Context, context.CancelFunc) {
+	return signal.NotifyContext(parent, windows.SIGTERM, windows.SIGINT)
```

Comment:
Not a review question, but does it actually work? How? Windows has signals now?

Reply:
I take a brief look in signal package docs. Docs said that we need to use os.Interrupt. After I've added it to the NotifyContext it started to work.  :smile: